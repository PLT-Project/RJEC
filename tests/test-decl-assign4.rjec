func main() {
  var i,j int = 42, 30;
  printi(i);
  printi(j);

  if true {
      j = 60;
      printi(j);
      var i int = 52;
      printi(i);
  }
  
  printi(i);
  printi(j);
}