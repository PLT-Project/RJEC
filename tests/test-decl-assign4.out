42
30
60
52
42
60